Sources/Main.cpp

```diff
@@ -26,6 +26,7 @@
 #include "Input.h"
 #include "About.h"
 #include "FileExport.h"
+#include <Clipbrd.hpp>
```

Comment:
This seems to be a non-relevant change. Please remove.

Reply:
Ok